### ISSUE TITLE
Frontend panic on protobuf

### ISSUE THREAD

**Issue by maximebuyse** [hacspec/hax#1014]
Running `cargo hax json` on https://github.com/stepancheg/rust-protobuf/tree/master panics with:

```
thread 'rustc' panicked at /home/maxime/cryspen/hax/frontend/exporter/src/types/copied.rs:1378:60:
DefId::expect_local: `DefId(22:6447 ~ protobuf[e121]::wire_format::WireType)` isn't local
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by W95Psp** [comment]
Thanks, that's a tiny bug, I have a fix, pushing a PR in a few minutes

**Comment by W95Psp** [comment]
Was fixed by #1015
